### ISSUE TITLE
Integrate YAML Front Matter to Preview

### ISSUE THREAD

**Issue by ketozhang** [shd101wyy/vscode-markdown-preview-enhanced#68]
It seems like MPE does support FrontMatter as seen on the Atom version ([Atom MPE Issue #100](https://github.com/shd101wyy/markdown-preview-enhanced/issues/100)).

Setting 

```json
"markdown.previewFrontMatter": "show"
```
fails to show the FrontMatter on MPE Preview.

Additionally is there anyway to show `title: MyTitle` as an H1 header? I would love have my MPE Preview as similar to my Jekyll site.

**Comment by shd101wyy** [comment]
You are changing the wrong settings. Search `frontMatter` and you will see.
![screenshot from 2018-01-08 00-53-09](https://user-images.githubusercontent.com/1908863/34661297-5ddf9b9a-f40e-11e7-8d7f-b4a5c21eedc1.png)

**Comment by ketozhang** [comment]
Yes I've done that as well. Either way it's set to `table` by default.

**Comment by ketozhang** [comment]
Ahh I've located the issue.  The frontMatter won't display if this `PandocParser` is set to true.

```json
"markdown-preview-enhanced.usePandocParser": true,
```
